tdrs-backend/tdpservice/core/admin.py

```diff
@@ -0,0 +1,66 @@
+"""Admin class for LogEntry objects."""
+from django.contrib import admin
+from django.contrib.admin.models import LogEntry, DELETION
+from django.utils.html import escape
+from django.urls import reverse
+from django.utils.safestring import mark_safe
+
+admin.site.unregister(LogEntry)
+
+
+@admin.register(LogEntry)
+class LogEntryAdmin(admin.ModelAdmin):
+    """Customize and restrict the LogEntry table in Django Admin."""
+
+    date_hierarchy = 'action_time'
+
+    list_filter = [
+        'user',
+        'content_type',
+        'action_flag'
+    ]
+
+    search_fields = [
+        'object_repr',
+        'change_message'
+    ]
+
+    list_display = [
+        'action_time',
+        'user',
+        'content_type',
+        'object_link',
+        'action_flag',
+    ]
+
+    def has_add_permission(self, request):
+        """Deny the user permission to create LogEntry objects in Django Admin."""
+        return False
+
+    def has_change_permission(self, request, obj=None):
+        """Deny the user permission to update LogEntry objects in Django Admin."""
+        return False
+
+    def has_delete_permission(self, request, obj=None):
+        """Deny the user permission to delete LogEntry objects in Django Admin."""
+        return False
+
+    def has_view_permission(self, request, obj=None):
+        """Only allow superusers to be able to view the LogEntry table."""
+        return request.user.is_superuser
+
+    def object_link(self, obj):
+        """Create a link to to corresponding objects for a given LogEntry."""
+        if obj.action_flag == DELETION:
```

Comment:
We don't have any tests that cover this line, through line 63, so we'll want to make sure that is covered

Reply:
Since we will never haver a DELETION for these objects, I can remove part of this logic too.